src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -139,8 +144,24 @@ public void addOrder(Person target, Order order) {
 
     @Override
     public List<Order> getOrders(Person target) {
-        addressBook.getOrders(target);
-        return target.getOrders();
+        requireNonNull(target);
+
+        return new ArrayList<>(target.getOrders());
+    }
+
+    @Override
+    public void deleteOrder(Person target, Order order) {
+        requireNonNull(target);
+        requireNonNull(order);
+        addressBook.deleteOrder(target, order);
+    }
+
+    @Override
+    public List<Order> getSortedOrders(Person person) {
+        requireNonNull(person);
+        List<Order> sortedOrders = new ArrayList<>(person.getOrders());
```

Comment:
I think you can make use of model.getOrders instead? And what is the rationale of creating a new instance of orders